regression/memory-analyzer/chain.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -e
+
+memory_analyzer=$1
+goto_gcc=$2
+name=${*:$#}
+name=${name%.exe}
+args=${*:3:$#-3}
+
+$goto_gcc -g -std=c11 -o "${name}.exe" "${name}.c"
+$memory_analyzer $args "${name}.exe"
```

Comment:
I'm not sure I'd use `.exe` when we're explicitly Unix-only. Probably `.bin`. But also not a big deal, might not be worth the trouble changing this.

Reply:
We use `.gb` in `goto-harness`. Changed.